CoreFoundation/CMakeLists.txt

```diff
@@ -358,6 +358,10 @@ else()
                                "/usr/include"
                                "/usr/include/libxml2")
 endif()
+target_include_directories(CoreFoundation
+                           PRIVATE
+                             ${ICU_INCLUDE_DIR})
+
```

Comment:
There is no reason to assume that `ICU_INCLUDE_DIR` is defined here.  This is built as an external project (since you can build CF outside of Foundation).  You need to replicate `FindICU.cmake` and add the appropriate `find_package`.

Reply:
I moved it into the Linux section and used `FindICU`. Although I think even if building CF outside of Foundation, ICU is still required as its CF that actually makes the ICU calls.

Reply:
Yeah, it is required.  I was just saying that you need the `find_package` if you are going to do this :-).